radiometric_normalization/normalize.py

```diff
@@ -17,26 +17,68 @@
     time_stack, pif, transformation, gimage
 
 
-def generate_transforms(candidate_path, reference_paths, config=None):
+def generate_transformations(candidate_path, reference_paths,
+                             config=None, pif_path=None):
     if config is None:
         config = {'time_stack_method': 'identity',
                   'pif_method': 'identity',
                   'transformation_method': 'linear_relationship'}
 
-    reference_image = time_stack.generate(
-        reference_paths,
-        method=config['time_stack_method'])
+    # Ensure all entries are there for partial configs
```

Comment:
What do you think of the approach of deleting generate_transformations and moving the (basic, whole workflow) code to a usage example for radiometric normalization? This would move the radiometric_normalization package toward being a library and would free us from having to build a sophisticated function for supporting permutation the user could desire.